tests/unit/command/test_studio.py

```diff
@@ -0,0 +1,91 @@
+from unittest import mock
+
+from dvc_studio_client.auth import AuthorizationExpired
+
+from dvc.cli import main
+from dvc.utils.studio import STUDIO_URL
+
+
+def test_studio_login_invalid_scope():
+    assert main(["studio", "login", "--scopes", "invalid!"]) == 1
```

Comment:
It looks like dvc is not responsible for checking this scope. Please move this to dvc-studio-client and remove it from here.